clang/lib/Driver/Driver.cpp

```diff
@@ -7093,6 +7095,15 @@ void Driver::BuildActions(Compilation &C, DerivedArgList &Args,
 
   handleArguments(C, Args, Inputs, Actions);
 
+  // If '-fintelfpga' argument is passed by users, add '-fsycl' option
+  // to the list of arguments, so that user's do not have to
+  // pass it explicitly in the commandline.
```

Comment:
```suggestion
  // If '-fintelfpga' is passed, add '-fsycl' to the list of arguments
```
Just a simplification